### PR TITLE
Upgrading project to latest stable ruby (2.6.3)

### PR DESCRIPTION
**Pull request by dtrenz**
https://www.ruby-lang.org/en/news/2019/04/17/ruby-2-6-3-released/

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.5.3
+2.6.3
```
Comment (jshier) [inline .ruby-version]: Please add the newline back to appease GitHub.

**Comment by jshier** [comment]
As part of this, can you also update the other mentioned versions of Xcode and the simulators in the Fastfile itself?

**Comment by dtrenz** [comment]
> As part of this, can you also update the other mentioned versions of Xcode and the simulators in the Fastfile itself?

I did this in a separate PR: #24

**Comment by dtrenz** [comment]
@jshier This PR should be good to go, now.

**Pull request merged**